monitoring/dr/alerts.test.yaml

```diff
@@ -0,0 +1,88 @@
+# These tests are meant to go hand-in-hand with the rendered alert rule.
+# Use github.com/scality/action-prom-render-test@python-renderer python module
+#
+# Render the alerts file with
+# gen-alert render alerts.yaml
+evaluation_interval: 1m
+rule_files:
+  - alerts.rendered.yaml
+
+tests:
+  - interval: 1m
+    input_series:
+      - series: up{drSinkInstance="", job="artesca-data-dr-source-base-queue-connector-metrics", namespace="zenko"}
```

Comment:
such serie does not exist, we have either (on the source):
```
up{job="artesca-data-dr-source-base-queue-connector-metrics", namespace="zenko"}
```

or (scraped via federation from the sink):
```
up{drSinkInstance="artesca-data-dr", job="artesca-data-dr-source-base-queue-connector-metrics", namespace="zenko"}
```

```suggestion
      - series: up{job="artesca-data-dr-source-base-queue-connector-metrics", namespace="zenko"}
```

(here, you define actual metrics series ; which is not exactly the same as the _selectors_ we use in alerts/dashboards to query metrics)